ts/Dashboards/Board.ts

```diff
@@ -796,6 +827,7 @@ namespace Board {
                 return Serializable
                     .fromJSON(JSON.parse(dashboardJSON)) as Board;
             } catch (e) {
+                // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
                 throw new Error(`${e}`);
```

Comment:
It is not good to overuse eslint exceptions in src. I know it isn't a line you wrote (I didn't look into the validity of this piece of code), but instead of adding the `eslint-disable`, we can do something like this:
```suggestion
                throw new Error('' + e);
```